doc/src/manual/interfaces.md

```diff
@@ -5,6 +5,79 @@ A lot of the power and extensibility in Julia comes from a collection of informa
 receive those functionalities, but they are also able to be used in other methods that are written
 to generically build upon those behaviors.
 
+## [Type Properties](@id man-type-properties)
```

Comment:
Properties

Reply:
What do you mean exactly? Just calling the section "Properties"?

I think that'd be confusing, because there are many types of properties, but just one type properties ;)

Reply:
Except that these are not "type" properties, they are *instance* properties.

Reply:
Why so? They do not vary from instance to instance, they are defined for all instances of that particular type, no?

Reply:
Nope, in general they can vary from instance to instance.

For example, in dataframes they depend on the instance.  Similarly for PyCall's `PyObject` type (for which the properties come from the Python object instance).

That's why `propertynames` takes the instance as the argument, not the type.

Reply:
Oh, I see! Thanks for pointing it out.

What shall we call the section, then? Just "Properties"? Or "Instance Properties"?

Reply:
I vote for something more explicit, like:
```
Overloading `object.field`: Instance Properties
```

Reply:
I have changed it now. The whole title seemed a bit long (more so when compared to other titles in this section), so I have added the "Overloading ..." part as a sub-subheader.

This is not super orthodox, but I think it could improve legibility. Is it legal to have only one sub-subheader?

Reply:
LarkAnspach appears to be a spam account, but carry-on

Reply:
Yeah, kinda noticed looking at his contributions. Never mind, it sparked a proper conversation.

I do not have anything to add, so it is up to you whether to suggest something else, or leave it like it is.